be/src/Project/Web/Seller/Seller.Web/Areas/Products/DomainModels/CategorySchema.cs

```diff
@@ -9,5 +9,6 @@ public class CategorySchema
         public string UiSchema { get; set; }
         public DateTime LastModifiedDate { get; set; }
         public DateTime CreatedDate { get; set; }
+        public string Language { get; set; }
```

Comment:
**suggestion (non-blocking):** Can you move higher? For example before lastModifiedDate